build/minio.sh

```diff
@@ -23,6 +23,7 @@ MINIO_CHART_VERSION="8.0.0"
 
 install_minio ()
 {
+    local pvc_size=${2:-"2Gi"}
```

Comment:
2Gi storage for a test seems way more than required. Can we run tests with a lesser minio storage capacity?

Reply:
reverted